test/common/testlib.py

```diff
@@ -246,6 +246,12 @@ def __init__(
         self.coverage_label = coverage_label
         self.machine = machine
 
+        # HACK: Tests which don't yet get along with real mouse clicks in Chromium
+        # can opt into falling back to the old MouseEvent emulation; this is cheating, but fixing
+        # all the tests at once is too much work. Remove this once all tests in all our projects
+        # got fixed.
+        self.chromium_fake_mouse = False
```

Comment:
So this means it can become a decorator which I think I would prefer as it feels more explicit.

Reply:
Nevermind, this is on the `Browser` class so would be hard to access in a decorator so nevermind.